website/projects/scimaps/src/app/pages/books/books.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { BooksComponent } from './books.component';
+
+describe('BooksComponent', () => {
+  let component: BooksComponent;
```

Comment:
Add proper tests! You could test that it creates an overview element for each book. I.e.
```
const { find } = render({ bind: { books: [one] } });
const overviews = find('sci-book-overview');
expect(overviews.length).toEqual(1);
```

Reply:
Sorry, forgot we were doing tests on this project